pkgs/applications/version-management/smartgithg/default.nix

```diff
@@ -33,7 +33,7 @@ stdenv.mkDerivation rec {
       which
     ];
     runtime_lib_paths = lib.makeLibraryPath [
-      gtk2 glib
+      gtk3 glib
```

Comment:
does this no longer depend on `gtk2`? (changelog?)

Reply:
There are some mentions of gtk 3 in the changelog but it seems to work with either version.
Changelog can be found at https://www.syntevo.com/smartgit/changelog.txt.